SwiftWisdomTests/StandardLibrary/Set/SetTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SetTests.swift
+//  SwiftWisdom
+//
+//  Created by Patrick Butkiewicz on 3/1/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+final class SetTests: XCTestCase {
```

Comment:
i think these tests are solid, but we should deprecate these. Set now implements Collection, and ExpressibleByArrayLiteral. Maybe the only not explicitly defined operator is ip_passes...be open to thoughts on this one.